### ISSUE TITLE
Building for Windows throws "undefined: syscall.Dup2" error

### ISSUE THREAD

**Issue by gotun** [bugsnag/panicwrap#14]
### Expected behavior
When building the application for Windows with bugsnag/panicwrap, the application builds without error.
```
GOOS=windows GOARCH=amd64 go build -o "app.exe"
```

### Observed behavior
The following error is raised
```
vendor/github.com/bugsnag/panicwrap/dup2.go:10:9: undefined: syscall.Dup2
```

### Steps to reproduce
Simply import bugsnag-go or panicwrap.

### Version
bugsnag/panicwrap v1.2.1 and v1.2.2

### Additional information
A recent change (see https://github.com/bugsnag/panicwrap/pull/11/files#diff-d60cb8d026bb18f93c915c1c38a670de9c8163d30ec1258ad95bb02fd30954ebL1) was made that allows dup2.go to be built for Windows. Adding `!windows` to line 1 of dup2.go appears to resolve the issue.

**Comment by yousif-bugsnag** [comment]
Hi @gotun

Thanks for the report and the suggested fix. We will look to fix this in a future version.

**Comment by kattrali** [comment]
This is fixed in [v1.3.1](https://github.com/bugsnag/panicwrap/releases/tag/v1.3.1), thanks for letting us know!
